src/AppInstallerCLICore/Workflows/ShellExecuteInstallerHandler.cpp

```diff
@@ -97,22 +100,29 @@ namespace AppInstaller::CLI::Workflow
                 installerArgs += ' ' + installerSwitches.at(ManifestInstaller::InstallerSwitchType::Language);
             }
 
-            // Construct install location arg if necessary.
-            if (context.Args.Contains(Execution::Args::Type::InstallLocation) && installerSwitches.find(ManifestInstaller::InstallerSwitchType::InstallLocation) != installerSwitches.end())
-            {
-                installerArgs += ' ' + installerSwitches.at(ManifestInstaller::InstallerSwitchType::InstallLocation);
-            }
-
             // Construct log path arg.
             if (installerSwitches.find(ManifestInstaller::InstallerSwitchType::Log) != installerSwitches.end())
             {
                 installerArgs += ' ' + installerSwitches.at(ManifestInstaller::InstallerSwitchType::Log);
             }
 
-            // Construct custom arg.
-            if (installerSwitches.find(ManifestInstaller::InstallerSwitchType::Custom) != installerSwitches.end())
+            if (isUpdate && installerSwitches.find(ManifestInstaller::InstallerSwitchType::Update) != installerSwitches.end())
+            {
+                installerArgs += ' ' + installerSwitches.at(ManifestInstaller::InstallerSwitchType::Update);
+            }
+            else

```

Comment:
>else [](start = 12, length = 4)

Why are the custom args excluded from update? Do we need to have an entire "update" switches block that can override the base ones?